### ISSUE TITLE
array is an invalid proptype for aria-label

### ISSUE THREAD

**Issue by NightJar** [silverstripe/silverstripe-admin#1391]
https://github.com/silverstripe/silverstripe-admin/blob/be31300d49ec28520e9ff3aed2ccf4b35efae759/client/src/components/Button/Button.js#L10

> Warning: Failed prop type: Invalid prop `aria-label` of type `array` supplied to `Button`, expected `string`.
checkPropTypes.js:20

The `children` prop is an array of all children. The [button element](https://html.spec.whatwg.org/multipage/form-elements.html#the-button-element)'s content model accepts all [phrasing content](https://html.spec.whatwg.org/multipage/dom.html#phrasing-content-2) except [interactive content](https://html.spec.whatwg.org/multipage/dom.html#interactive-content). The children prop could rightfully contain components/elements in this context.

**Comment by GuySartorelli** [comment]
Is this actually causing any problems? If so can you please describe what problems this is causing?

**Comment by NightJar** [comment]
Errors in the console whenever this widget is used.
